x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LocalPhysicalPlanOptimizer.java

```diff
@@ -194,6 +196,8 @@ private static Set<Attribute> missingAttributes(PhysicalPlan p) {
     }
 
     public static class PushFiltersToSource extends OptimizerRule<FilterExec> {
+        private static int HALF_FLOAT_MAX = 65504;
```

Comment:
I did not find this constant defined in the code base - anyone know if it is defined somewhere, after all?